src/overlays/actors/ovl_En_Heishi2/z_en_heishi2.h

```diff
@@ -19,11 +19,11 @@ typedef struct EnHeishi2 {
     /* 0x0266 */ char unk_266[0x06];
     /* 0x026C */ Vec3s unk_26C; // padding inbetween these
     /* 0x0274 */ Vec3f unk_274; 
-    /* 0x0280 */ Vec3f unk_280; // camera related
-    /* 0x028C */ Vec3f unk_28C; // camera related
-    /* 0x0298 */ Vec3f unk_298; // camera related
+    /* 0x0280 */ Vec3f subCamEye;
+    /* 0x028C */ Vec3f subCamAt;
+    /* 0x0298 */ Vec3f subCamAtInit;
```

Comment:
Iirc some struct members like `subCamAtInit` `subCamEyeInit` are called `...Copy` in another place

Reply:
Yeah I realized that too very recently, they should be changed to `Init` now